.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -44,8 +44,8 @@ data class CIBuildModel (
                     functionalTests = listOf(
                             TestCoverage(TestType.quickFeedbackCrossVersion, OS.linux, JvmVersion.java8),
                             TestCoverage(TestType.quickFeedbackCrossVersion, OS.windows, JvmVersion.java8),
-                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java10),
-                            TestCoverage(TestType.parallel, OS.linux, JvmVersion.java8, JvmVendor.ibm))
+                            TestCoverage(TestType.parallel, OS.linux, JvmVersion.java10),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java8, JvmVendor.ibm))
```

Comment:
Shall we still try to fix the problems with parallel test execution and IBM JDK 8?

@oehme Shall we drop test coverage for IBM JDK altogether? Can we look at our statistics how many people are using IBM JDK to run Gradle?

Reply:
I really doubt if we can fix it. According to the error:

```
Caused by: java.lang.NullPointerException
	at java.io.File.<init>(File.java:288)
	at java.lang.Package$1.run(Package.java:578)
	at java.lang.Package$1.run(Package.java:571)
	at java.security.AccessController.doPrivileged(AccessController.java:640)
	at java.lang.Package.defineSystemPackage(Package.java:571)
	at java.lang.Package.getSystemPackages(Package.java:562)
	at java.lang.VMAccess.getSystemPackages(VMAccess.java:154)
	at com.ibm.oti.vm.BootstrapClassLoader.getPackages(BootstrapClassLoader.java:100)
	at java.lang.ClassLoader.getPackages(ClassLoader.java:1205)
	at java.lang.ClassLoader.getPackages(ClassLoader.java:1208)
	at worker.org.gradle.internal.classloader.FilteringClassLoader$RetrieveSystemPackagesClassLoader.getPackages(FilteringClassLoader.java:63)
```

This is out of our control. I suspect under certain conditions [this issue](http://www-01.ibm.com/support/docview.wss?uid=swg1IJ00181) is triggered again. I'm fine with not testing IBM JDK completely.

Reply:
@eriwen would have the statistics.

IBM JDK is now called OpenJ9 and hosted at the Eclipse foundation. Maybe we should test that in a more recent version instead.

Reply:
I just realized we're not using latest IBM JDK - latest is `8.0.5.21` but we're using `8.0.5.20`. I just upgraded, let's keep this pr open and see if situation gets improved.

Reply:
It doesn't seem to be improved at all after the upgrading: https://github.com/gradle/gradle-private/issues/1491

I don't know if this is caused by parallel execution, I'd say let's first make it non-parallel, then remove test coverage for IBM JDK totally if it still occurs.

Reply:
Have you considered what I proposed above?

Reply:
@oehme You mean "more recent version"? The version we're currently testing is released on Sep 3, 2018: https://developer.ibm.com/javasdk/downloads/sdk8/

Are you saying we try OpenJ9 latest released on Aug 14, 2018 https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=openj9 ?